.github/workflows/dockerhub.yml

```diff
@@ -7,46 +7,52 @@
 #    DOCKERHUB_REPO
 # This workflow will use them automatically.
 
-# FIXME: Refactor this workflow using matrix and the new docker args
-
 name: Publish Docker images
 
-on: workflow_dispatch
-#    TODO: put this back once the docker files work correctly
-#  release:
-#    types: [released]
+on:
+    workflow_dispatch:
+    push:
+        tags:
+            - 'v*'
```

Comment:
When I created the workflow, I was a bit hesitant to use a required tag format, as it was not 100% established flow.. And I'm still a bit hesitant...although, I consider this as good practice. jfyi - We should 
1. describe the exact deployment flow in the docs
2. Maybe introduce a bash/batch script `release.sh --minor` or something for consistent runs